styles.css

```diff
@@ -110,6 +154,39 @@ p {
   padding-bottom: 32%;
 }
 
+@media screen and (min-width: 800px) {
+  .section {
+    background-image: url(/images/Group\ 2.png);
+    background-position: right;
+    background-repeat: no-repeat;
+    height: calc(100vh - 110px);
+  }
```

Comment:
As specified in the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/f9250f0f5b9b93daa778a2e480c689b691c5d434/portfolio/4_desktop_version.md?plain=1#L41), kindly ensure that the website displays properly on desktop sizes (768px or wider)

Your design at 808px:
![image](https://user-images.githubusercontent.com/20554583/206877607-fddc97e0-4f91-478e-86aa-cbaee81c0432.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/206877567-90ce2971-4e66-4b81-9a7a-e71a8f44d64f.png)